src/adapters/realm.ts

```diff
@@ -16,7 +16,7 @@ export const CATALYST_NAME_CONFIG_FILE = '.catalyst-name'
 
   1. If there is a file .catalyst-name, just use the name there
   2. If the local lighthouse has a name, pick it, store it in .catalyst-name for next time
```

Comment:
How can we rephrase this so it doesn't say lighthouse?

Reply:
well, is still the lighthose env var